src/Resources/skeleton/validator/Validator.tpl.php

```diff
@@ -9,7 +9,7 @@ class <?= $class_name ?> extends ConstraintValidator
 {
     public function validate($value, Constraint $constraint)
     {
-        /* @var $constraint \<?= $constraint_class_name ?> */
+        /** @var \<?= $constraint_class_name ?> $constraint */
```

Comment:
This was the most problematic, because phpstan kept failing on the generated validator classes and had to manually fix the typehints